app.config.js

```diff
@@ -1,7 +1,28 @@
+const webpack = require('webpack')
+const IgnoreNotFoundExportPlugin = require('ignore-not-found-export-webpack-plugin')
+
+const appConfig = {
+  plugins: [
+    new webpack.ProvidePlugin({
+      'cozy.client': 'cozy-client-js/dist/cozy-client.js',
+      'cozy.bar': 'cozy-bar/dist/cozy-bar.js'
+    }),
+    new IgnoreNotFoundExportPlugin({
+      include: [
+        /@bitwarden\/jslib\/services\/crypto\.service/, // EEFLongWordList
+        /@bitwarden\/jslib\/services\/passwordGeneration\.service/, // EEFLongWordList
+        /WebVaultClient/, // EEFLongWordList
+        /cozy-ui\/transpiled\/react\/Portal\/index\.js/ // preact-portal
+      ]
+    })
```

Comment:
I think it might be good to add a comment explaining what this does, as the plugin docs are not very explicit. At first I thought these were modules we tried to import but failed, but I suppose it's actually a list of modules that import something (eg. `EEFLongWordList `) that can't be found?
If it suppresses all warnings, isn't there a risk to miss a missing export that might be more important?

Reply:
I've added a comme
nt to clarify.

> If it suppresses all warnings, isn't there a risk to miss a missing export that might be more important?

We list all files that would generate a warning so I think we are safe. 

And the warnings here come from node_modules; from the point of view of the home, it makes sense to ignore warnings that should be dealt with at the library level.